hyperspy/drawing/_widgets/range.py

```diff
@@ -18,13 +18,18 @@
 
 import inspect
 import logging
+from packaging.version import Version
 
+import matplotlib
 import numpy as np
 
 from hyperspy.drawing.widget import ResizableDraggableWidgetBase
 from hyperspy.defaults_parser import preferences
 
-from hyperspy.external.matplotlib.widgets import SpanSelector
+if Version(matplotlib.__version__) >= Version('3.6.0'):
+    from matplotlib.widgets import SpanSelector
+else:
+    from hyperspy.external.matplotlib.widgets import SpanSelector
```

Comment:
Would it make sense to add a comment in `external/matplotlib/widgets.py` that once the minimum supported version of Matplotlib should be set to `3.6.0`, the function `SpanSelector` could be deleted? But I guess it is clear enough from this call?

Reply:
There is already this comment: https://github.com/hyperspy/hyperspy/blob/cf75ac8440dc146e678a985128a62d92a65dd574/hyperspy/external/matplotlib/widgets.py#L1-L2